content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -90,9 +90,8 @@ If you look in the sample services, you can see that the productpage application
 def getForwardHeaders(request):
     headers = {}
 
-    user_cookie = request.cookies.get("user")
-    if user_cookie:
-        headers['Cookie'] = 'user=' + user_cookie
+    if 'user' in session:
+        headers['end-user'] = session['user']
```

Comment:
These changes accurately reflect the code that is in the productpage service in the `release-1.0` branch.